unit_tests/test_nemesis.py

```diff
@@ -223,3 +223,34 @@ def test_use_disabled_monkey():
     collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
 
     assert 'disrupt_toggle_table_gc_mode' in collected_disrupt_methods_names
+
+
+def test_list_all_available_nemesis(export=False):
+    import pickle
+
+    tester = FakeTester()
+
+    tester.params["nemesis_seed"] = '1'
+    tester.params["nemesis_selector"] = []
+    sisyphus = SisyphusMonkey(tester, None)
+
+    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisyphus.disruptions_list}
+
+    assert len(collected_disrupt_methods_names) == 67
+
+    if export:
+        # Export the list of nemesis to a static file for later usage
+        with open("test_data/nemesis_list", "wb") as fp:
```

Comment:
the location of this file, shouldn't be in the unittest test data, but in `data_dir`

also we should consider having the path as part of the `SisyphusMonkey` class, and can be overwritten, for example for siren-tests, which have a bit different list  and flags, since it has more Nemesis